ui/pages/confirm-send-ether/confirm-send-ether.container.js

```diff
@@ -19,21 +19,23 @@ const mapDispatchToProps = (dispatch) => {
   return {
     editTransaction: (txData) => {
       const { id, txParams } = txData;
-      const { from, gas: gasLimit, gasPrice, to, value: amount } = txParams;
-
+      const {
+        from,
+        gas: gasLimit,
+        gasPrice,
+        to: address,
+        value: amount,
+      } = txParams;
       dispatch(
-        updateSend({
-          from,
+        editTransaction({
+          id: id?.toString(),
```

Comment:
Do we allow editing transactions without an `id`?

Reply:
No, but as the previous version of the code used this syntax I kept it to be safe instead of changing the behavior too drastically (like throwing an error here if the id is missing or introducing new UI for when this case happens)